contracts/ocr2/src/integration_tests.rs

```diff
@@ -801,3 +801,68 @@ fn set_link_token() {
         .unwrap();
     assert_eq!(token_addr, new_link_token);
 }
+
+#[test]
+fn set_billing_payout() {
+    let mut env = setup();
+    let deposit = Decimal::from_str("1000").unwrap().0;
+    // expected in juels
+    let observation_payment = Uint128::from(5 * GIGA);
+    let reimbursement = Decimal::from_str("0.001871716").unwrap().0;
+
+    // -- set billing
+    // price in uLUNA
+    let recommended_gas_price = Decimal::from_str("0.01133").unwrap();
+    let msg = ExecuteMsg::SetBilling {
+        config: Billing {
+            recommended_gas_price_micro: recommended_gas_price,
+            observation_payment_gjuels: 5,
```

Comment:
Consider changing to some different/new billing values here.

Reply:
resolved the changes during the rebase fixes